src/app/clusters/door-lock-server/door-lock-server.h

```diff
@@ -95,12 +96,16 @@ class DoorLockServer
      * @param endpointId ID of the endpoint to the lock state
      * @param newLockState new lock state
      * @param opSource source of the operation (will be used in the event).
+     * @param fabricIdx fabric index
+     * @param nodeId node id
```

Comment:
Again, better to more clearly document which fabric and node these are: those that are responsible for operating the lock.

And maybe we should test that those are only non-null for the "remote" optype?  Or at least document here that they must be provided for that optype.

Reply:
added error log for that case now